### PR TITLE
Added DSC Windows Service template

### PR DESCRIPTION
**Pull request by dunedinsoftware**
--- Efficiently align Windows services to a desired state (stopped/started)

### Step template checklist

- [X] `Id` should be a **GUID** that is not `00000000-0000-0000-0000-000000000000`
- [X] `Version` should be incremented, otherwise the integration with Octopus won't update the step template correctly
- [X] Parameter names should not start with `$`
- [X] `LastModifiedBy` field must be present, and (_optionally_) updated with the correct author
- [X] If a new `Category` has been created:
   - [X] An image with the name `{categoryname}.png` must be present under the `step-templates/logos` folder
   - [X] The `switch` in the `humanize` function in [`gulpfile.babel.js`](https://github.com/OctopusDeploy/Library/blob/master/gulpfile.babel.js#L92) must have a `case` statement corresponding to it

### CODE REVIEW

File: step-templates/dsc-windows-service.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      "Label": "Timeout in seconds",
+      "HelpText": "The number of seconds to wait for the service/s to align before timing out and throwing an exception",
+      "DefaultValue": "300",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    }
+  ],
+  "LastModifiedOn": "2017-06-20T14:00:00.000+00:00",
+  "LastModifiedBy": "alexhildyard",
```
Comment (alfhenrik) [inline step-templates/dsc-windows-service.json]: If you could update this to **dunedinsoftware** then the author link on the Library site will link correctly to your :octocat: profile

File: step-templates/dsc-windows-service.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  "$Meta": {
+    "ExportedAt": "2017-06-20T14:35:09.389Z",
+    "OctopusVersion": "3.1.5",
+    "Type": "ActionTemplate"
+  }
```
Comment (alfhenrik) [inline step-templates/dsc-windows-service.json]: Can you please add a "Category" property here with a value of "windows" that be great as it'll get the correct icon on the Library site and also inside Octopus.

**Comment by dunedinsoftware** [comment]
Changes made as requested; CLA signed: https://cla-assistant.io/OctopusDeploy/Library?pullRequest=545

Thanks again,

Alex

**Pull request merged**